ScnSoft-Education/LeetCodeTraining/Program.cs

```diff
@@ -1,2 +1,2 @@
 ﻿// See https://aka.ms/new-console-template for more information
```

Comment:
Please revert the changes in this file if it is possible.
Changes in it should be in this PR.

Reply:
Ok, leave this changes but try to not add not related to the PR changes.
Please do them in main branch if it is required.